src/components/Tooltip.tsx

```diff
@@ -83,7 +83,7 @@ function Popper({ triggerRef, content, placement = "auto" }: PopperProps) {
       ref={popperRef}
       style={styles.popper}
       {...attributes.popper}
-      css={Css.maxw("320px").bgGray900.white.px1.py("4px").br4.xs.z999.$}
+      css={Css.maxw("320px").bgGray900.white.px1.py("4px").br4.xs.add("zIndex", 999999).$}
```

Comment:
**question:** What is the z-index of the button modal?

Reply:
Discovered it is `100000`. This is needed mostly because we use `Popper` right now instead of React-Aria. One day we'll move to not need popper. For now this is fine.